### PR TITLE
Update csrankings.csv

### PR DESCRIPTION
**Pull request by kroegern1**
Added 2, updated 1, University of Florida

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -757,7 +757,8 @@ Alin Dobra,University of Florida,http://www.cise.ufl.edu/~adobra,smEOOBsAAAAJ
 Alin Olimpiu Albu-Schäffer,TU Munich,https://www.professoren.tum.de/en/albu-schaeffer-alin,yP_zL3gAAAAJ
 Alina Ene,Boston University,http://www.bu.edu/cs/alina-ene-2,v1_kfR8AAAAJ
 Alina Oprea,Northeastern University,http://www.ccs.neu.edu/home/alina,16J3izoAAAAJ
-Alina Sirbu,University of Pisa,http://kdd.isti.cnr.it/people/s%C3%AErbu-alina,o-G11bQAAAAJ
+Alina Sirbu,University of Pisa,http://kdd.isti.cnr.it/people/s%C3%AErbu-alina,o-G11bQAAAAJ
+Alina Zare,University of Florida,https://faculty.eng.ufl.edu/machine-learning/,OviqAJsAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not in computer science.
https://www.cise.ufl.edu/people/faculty/

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please remove the non-CS faculty member, who does not appear to meet the inclusion criteria (not in CS, not an affiliate).

File: csrankings.csv (hunk 1 above)
Comment (kroegern1) [reply csrankings.csv]: Alina Zare has graduate faculty status in the computer science department.

Comment (emeryberger) [reply csrankings.csv]: Can you point to evidence of this?

**Comment by emeryberger** [comment]
Bump @kroegern1

File: csrankings.csv (hunk 1 above)
Comment (kroegern1) [reply csrankings.csv]: Sorry, I don't see proof online, but I know first-hand.

Comment (emeryberger) [reply csrankings.csv]: That's not enough, sorry. The most expedient way to handle this is to have the faculty member indicate this publicly on this thread.

**Comment by emeryberger** [comment]
Bump @kroegern1

**Pull request merged**